neural_modelling/src/spike_source/poisson/spike_source_poisson.c

```diff
@@ -350,22 +390,13 @@ static bool initialize(void) {
         return false;
     }
 
-    if (!read_poisson_parameters(
-            data_specification_get_region(POISSON_PARAMS, ds_regions))) {
+    if (!read_rates(
+            data_specification_get_region(RATES, ds_regions))) {
         return false;
     }
 
-    // Loop through slow spike sources and initialise 1st time to spike
-    for (index_t s = 0; s < params.n_spike_sources; s++) {
-        spike_source_t *p = &poisson_parameters[s];
-        if (!p->is_fast_source) {
-            p->time_to_spike_ticks =
-                    slow_spike_source_get_time_to_spike(p->mean_isi_ticks);
-        }
-    }
-
     // print spike sources for debug purposes
-    print_spike_sources();
+    // print_spike_sources();
```

Comment:
Again probably uncomment this and put it in a LOG_DEBUG if statement.

Reply:
Done.  Also removed some other weirdnesses.